### ISSUE TITLE
Access to sass files from npm/yarn package

### ISSUE THREAD

**Issue by Soullivaneuh** [swagger-api/swagger-ui#4966]
If I add the package:

```
yarn add swagger-ui
```

I only have this:

![image](https://user-images.githubusercontent.com/1698357/47487394-a9063e80-d842-11e8-95f1-4ffa994fe588.png)

I need the sass file to override the variables. Could you please add this possibility?

**Comment by Soullivaneuh** [comment]
Current workaround: Adding the tar.gz of the release:

```
yarn add https://github.com/swagger-api/swagger-ui/archive/v3.19.4.tar.gz
```

But you wont get any update anymore.

**Comment by Soullivaneuh** [comment]
Also, a documentation part about how to use sass files would be great. :+1:

**Comment by soullivaneuh** [comment]
Any feedback about this issue? :thinking:

**Comment by shockey** [comment]
Yep! We've avoided this until now because our SASS variable strategy (and CSS patterns in general) aren't in _great_ shape, and we don't want folks to start relying on what naming conventions we're using at the moment (related: https://github.com/swagger-api/swagger-ui/issues/4460).

Would exposing SCSS in the module be acceptable if we provided it outside of our semantic versioning contract, meaning that future minor and patch versions could break SCSS variable names and values?

**Comment by soullivaneuh** [comment]
> if we provided it outside of our semantic versioning contract, meaning that future minor and patch versions could break SCSS variable names and values?

@shockey It would be better to have BC rules on it as we use it for overriding.

However, if this is too cumbersome for you, this may be acceptable if a warning message about that is present on the README. :+1:

**Comment by soullivaneuh** [comment]
@shockey Friendly ping. :-)

**Comment by jpoehnelt** [comment]
> Would exposing SCSS in the module be acceptable if we provided it outside of our semantic versioning contract, meaning that future minor and patch versions could break SCSS variable names and values?

Absolutely. Right now, I'll be seeking out a workaround that is more likely to break between versions.

**Comment by Drafter500** [comment]
This is highly needed. I see it as the only normal way to co customise the theme
